post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-bare-metal.adoc

```diff
@@ -1,16 +1,20 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: creating-multi-arch-compute-nodes-bare-metal
 [id="creating-multi-arch-compute-nodes-bare-metal"]
-= Creating a cluster with multi-architecture compute machine on bare metal
+= Creating a cluster with multi-architecture compute machines on bare metal, {ibm-power-title}, or {ibm-z-title}
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To create a cluster with multi-architecture compute machines on bare metal, you must have an existing single-architecture bare metal cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user provisioned cluster on bare metal]. You can then add 64-bit ARM compute machines to your {product-title} cluster on bare metal.
+To create a cluster with multi-architecture compute machines on bare metal (`x86_64`), {ibm-power-name} (`ppc64le`), or {ibm-z-name} (`s390x`) you must have an existing single-architecture cluster on one of these platforms. Follow the installations procedures for your platform:
```

Comment:
@prb112 Does adding "either" make it more clear? 
```suggestion
To create a cluster with multi-architecture compute machines on either bare metal (`x86_64`), {ibm-power-name} (`ppc64le`), or {ibm-z-name} (`s390x`) you must have an existing single-architecture cluster on one of these platforms. Follow the installations procedures for your platform:
```

Reply:
Yes. That's much clearer.